### PR TITLE
Fix getblocksubsidy suggestions

### PR DESCRIPTION
**Pull request by daira**
This is a follow-up to #6963.

### CODE REVIEW

**Comment by daira** [comment]
Only the last three commits are relevant, because this is targeted for after the merge of the `release-v6.0.0` branch back to `master`.

File: src/consensus/params.cpp [hunk 1]
```diff
@@ -372,16 +376,10 @@ namespace Consensus {
 
         // Funding streams are disabled if Canopy is not active.
         if (NetworkUpgradeActive(nHeight, Consensus::UPGRADE_CANOPY)) {
```
Comment (daira) [inline src/consensus/params.cpp:378]: This isn't strictly necessary; `GetActiveFundingStreams(nHeight)` will be empty if Canopy is not enabled.

**Comment by daira** [comment]
`Params::GetActiveFundingStreamElements` is used in consensus code so this requires three reviews.

**Review by nuttycom — APPROVED** [review]
utACK 4ddd3405c58165dcea6aadf2a784c939e54f9cf7

**Review by str4d — APPROVED** [review]
utACK 4ddd3405c58165dcea6aadf2a784c939e54f9cf7

**Pull request merged**